app/scripts/views/mixins/password-strength-mixin.js

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'lib/promise',
+  'lib/runtime-require',
+  'lib/url'
+], function (p, runtimeRequire, Url) {
+  'use strict';
+
+  var PasswordStrengthMixin = {
+    initialize: function (options) {
+      this._able = options.able;
```

Comment:
Do we need this `initialize` and `options`? I don't see it being called anywhere. Am I missing something?